modules/ossm-rn-deprecated-features.adoc

```diff
@@ -26,10 +26,3 @@ The following custom resources are deprecated in this release and will be remove
 ** `ClusterRbacConfig`(versions prior to {ProductName} 1.0)
 ** `ServiceMeshRbacConfig` ({ProductName} version 1.0 and later)
 * In Kiali, the `login` and `LDAP` strategies are deprecated. A future version will introduce authentication using OpenID providers.
-
-The following components are also deprecated in this release and will be replaced by the *Istiod* component in a future release.
```

Comment:
Should this be removed? I think this is still relevant in 1.x.

Reply:
This file is for 2.0.  That line is still there in the ossm-rn-deprecated-features-v1x.adoc file.  (I had to make at least one or two edits to make sure the 1.x and 2.x release notes published something different to make sure things were working.)